util/expensivequery/expensivequery.go

```diff
@@ -49,6 +54,11 @@ func (eqh *Handle) SetSessionManager(sm util.SessionManager) *Handle {
 // Run starts a expensive query checker goroutine at the start time of the server.
 func (eqh *Handle) Run() {
 	threshold := atomic.LoadUint64(&variable.ExpensiveQueryTimeThreshold)
+	systemMem, err1 := memory.MemTotal()
+	if err1 != nil {
+		logutil.BgLogger().Warn("Get system memory fail.", zap.Error(err1))
```

Comment:
What would happen If this error occurs?
And should it be suitable for k8s environment?

Reply:
TiDB can't get the system memory in k8s environment, so we need add a config and set it manually... PTAL again